gpu_bdb/bdb_tools/utils.py

```diff
@@ -960,35 +960,33 @@ def train_clustering_model(training_df, n_clusters, max_iter, n_init):
     given dataframe and returns the resulting
     labels and WSSSE"""
 
-    from cuml.cluster.kmeans import KMeans
-
-    best_sse = 0
-    best_model = None
+    from cuml.cluster.kmeans import KMeans as cuKMeans
+    from sklearn.cluster import KMeans
 
     # Optimizing by doing multiple seeding iterations.
-    for i in range(n_init):
-        model = KMeans(
```

Comment:
Why did we get ride of this for loop  .  That seems like a major logic change !!!!!

Reply:
Might be the cause of the deviation in cluster quality we are seeing.

Reply:
Thanks for the offline explanation @ChrisJar ,  I think we can let this be then. 

Discussion
```
 I think the n_init argument performs the multiple centroid seeds:

“n_init: Number of time the k-means algorithm will be run with different centroid seeds. 

The final results will be the best output of n_init consecutive runs in terms of inertia.”
```